assets/app/scripts/directives/util.js

```diff
@@ -0,0 +1,11 @@
+angular.module('openshiftConsole')
+  .directive('selectOnFocus', function() {
+    return {
+      restrict: 'A', 
+      link: function($scope, element, attrs) {
+        $(element).focus(function () {
```

Comment:
double check in different browsers to make sure this works on keyboard focus and on click